assets/src/edit-story/components/panels/videoAccessibility.js

```diff
@@ -0,0 +1,137 @@
+import { useCallback, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { rgba } from 'polished';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Media, Row, TextInput } from '../form';
+import { SimplePanel } from './panel';
+import getCommonObjectValue from './utils/getCommonObjectValue';
+import getCommonValue from './utils/getCommonValue';
+
+const BoxedTextInput = styled(TextInput)`
+  padding: 6px 6px;
+  border-radius: 4px;
+`;
+
+const ExpandedTextInput = styled(BoxedTextInput)`
+  flex-grow: 1;
+`;
+
+const Note = styled.span`
+  color: ${({ theme }) => rgba(theme.colors.fg.v1, 0.54)};
+  font-family: ${({ theme }) => theme.fonts.body1.family};
+  font-size: 12px;
+  line-height: 16px;
+`;
+
+function VideoAccessibilityPanel({ selectedElements, onSetProperties }) {
+  const resource = getCommonValue(selectedElements, 'resource');
+  const { posterId, poster, title, alt } = getCommonObjectValue(
+    selectedElements,
+    'resource',
+    ['posterId', 'poster', 'title', 'alt'],
+    false
+  );
+  const [state, setState] = useState({ posterId, poster, title, alt });
+  useEffect(() => {
+    setState({ posterId, poster, title, alt });
+  }, [posterId, poster, title, alt]);
+
+  const handleSubmit = (evt) => {
+    onSetProperties(state);
+    evt.preventDefault();
+  };
+
+  const handleChangeImage = (image) => {
+    const newState = {
+      posterId: image.id,
+      poster: image.sizes?.medium?.url || image.url,
+    };
+    setState({ ...state, ...newState });
+    onSetProperties({ resource: { ...resource, ...newState } });
+  };
+
+  const handleChange = useCallback(
+    (property) => (value) => {
+      const newState = {
+        [property]: value,
+      };
+      setState({ ...state, ...newState });
+      onSetProperties({ resource: { ...resource, ...newState } });
+    },
+    [state, onSetProperties, resource]
+  );
+
+  return (
+    <SimplePanel
+      name="videoAccessibility"
+      title={__('Accessibility', 'web-stories')}
+      onSubmit={handleSubmit}
+    >
+      <Row>
+        <Media
+          value={state.poster}
+          onChange={handleChangeImage}
+          title={__('Select as video poster', 'web-stories')}
+          buttonInsertText={__('Set as video poster', 'web-stories')}
+          type={'image'}
+        />
+      </Row>
+      <Row>
+        <ExpandedTextInput
+          placeholder={__('Title', 'web-stories')}
+          value={state.title || ''}
+          onChange={handleChange('title')}
+          clear
+        />
+      </Row>
+      <Row>
+        <ExpandedTextInput
+          placeholder={__('Assistive text', 'web-stories')}
+          value={state.alt || ''}
+          onChange={handleChange('alt')}
+          clear
+        />
+      </Row>
+      <Row>
+        <Note>
+          {__('Enter an address to apply a 1 or 2 tap link', 'web-stories')}
```

Comment:
Copy/paste issue?

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/652/commits/0f570ebc94f3f3b1b1267a4bd076387590611813